test/k8sT/manifests/client.yaml

```diff
@@ -8,7 +8,7 @@ metadata:
 spec:
   containers:
   - name: app-frontend
-    image: byrnedo/alpine-curl
+    image: cilium/connectivity-container
```

Comment:
Please specify a version (not latest) for this image and write it here

Reply:
Fixed.